.travis.yml

```diff
@@ -34,6 +43,9 @@ install:
   - /usr/libexec/java_home -v 1.7 -F -V
   - java -Xmx32m -version && javac -J-Xmx32m -version
   - pushd j2objc-gradle
+  - echo For branch $TRAVIS_BRANCH
+  # Update the plugin submodule to latest HEAD if in the appropriate branch
+  - if [[ $TRAVIS_BRANCH == pluginHead ]]; then git pull origin master; fi
```

Comment:
Yeah let's just keep it simple.  Only one branch (master) that follows the latest release.  Frankly when we make a j2objc-config release, we can just do the update to the submodule.  That way the release is persistent, even outside Travis.  However, we can start that policy only with 0.5.0 as 0.4.2 (or whatever) lacks needed dependency code.

Btw, the tag sorting script fails when multiple digits are involved.  What you really want is:
`git tag -l --sort=version:refname v\* | tail -n 1` which was made for this